include/cc_useful.h

```diff
@@ -0,0 +1,40 @@
+//  Copyright (c) 2021-22, VMware Inc, and the Certifier Authors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef __CC_USEFUL_H__
+#define __CC_USEFUL_H__
```

Comment:
Introduced a new `.h` to keep useful stuff, that will be helpful elsewhere, too.

Reply:
@jlmucb - I have updated the corresponding `*.mak` files for these programs to include the `CC` for this new `.cc` file.

I think they should build fine.

I just don't have a way to test that these programs will, indeed, still build and run cleanly. 

So, for now, ... I'll merge this and we'll have to watch out for fallout next week.